Expensify/App
# Expand web TextInput as the user types

Fixes https://github.com/Expensify/ReactNativeChat/issues/103

### Tests
1. Type in the textbox on web, android, and iOS, confirm the textbox grows as you type
2. Delete text in the textbox using backspace, confirm the textbox resizes as you delete
3. Paste a chunk of text or delete a chunk of text, confirm the textbox resizes properly
